packages/ui/src/Alert.module.scss

```diff
@@ -0,0 +1,153 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+// TODO: potential max-height
+.alert {
+  display: inline-block;
+  border-radius: c.$borderRadius;
+  padding: c.$grid * 2.5;
+  width: c.$alertWidth;
+
+  .header {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+    .icon {
+      margin-right: c.$grid * 2.5;
+    }
+
+    .title {
+      margin-right: c.$grid * 8.5;
+
+      @include c.typographyHighlightFont;
+      @include c.typographyBodyNormal;
+      @include h.overflowEllipsis;
+    }
+
+    .close {
+      position: absolute;
+      right: 0;
+
+      &:hover,
+      &.hover {
+        background-color: transparent;
+        box-shadow: none;
+      }
+    }
+  }
+
+  /* Theme - Success */
+  &.success {
+    color: c.$colorNeutralWhite;
+    background-color: c.$colorSuccess;
+
+    .icon {
+      color: c.$colorNeutralWhite;
+    }
+
+    .closeIcon {
```

Comment:
If we went along with https://github.com/hazelcast/frontend-shared/pull/49/files#r511764453 we could probably remove this class at all

Reply:
Can't get the link working properly. Could you post a snippet of what do you mean?

Reply:
It was linking to the idea of using `color: inherit` for the transparent button.

Reply:
Was able to get rid of the class in the end. Addressed in: https://github.com/hazelcast/frontend-shared/pull/49/commits/87c15d54aaccef3c99a0047b1eae2ec54c719544